bench/wtperf/wtperf_opt.i

```diff
@@ -171,9 +171,8 @@ DEF_OPT_AS_UINT32(scan_table_count, 0,
 DEF_OPT_AS_CONFIG_STRING(sess_config, "", "session configuration string")
 DEF_OPT_AS_UINT32(session_count_idle, 0, "number of idle sessions to create. Default 0.")
 DEF_OPT_AS_CONFIG_STRING(table_config,
-  "key_format=S,value_format=S,type=lsm,exclusive=true,"
```

Comment:
Please add a comment here explaining that the defaults match the values used by MongoDB for collections.

Reply:
done

Reply:
If you changed the default here, then why did you need/want to add in all the `table_config` calls to the 500m files? Wouldn't this change alone be sufficient?

Reply:
The default "type" changed from lsm to file. These three 500m configurations do not set a type. I thought "type=file" suits these configs, but I could be wrong. These configs failed to run with or without my changes, WT-6084 would get into fixing them.